README.md

```diff
@@ -1,2 +1,83 @@
+
+## Mechanical Aspects of Design:
+<p align="justify">
+1)	3 servo motors have been used for the project. The function of the servo motor placed along the side face of bug’s body rotates the bevel gears which is turn raise the arm up. <br/>
+2)	Once the arms are raised, the 2 servo motors being attached on the arms(1 on each) then moves the arms forward. <br/>
+3)	Hence first the arms are raised, then they move forward, touch down and then bot moves forward. <br/>
+4)	A castor wheel is also attached so that the bot can move smoothly without any restrictions due to friction. <br/> 
+
+
+## Electronic Aspects of Design:
+<p align="justify">
+1)	Arduino has been attached to the body which controls the servo motors’ performance like angle of rotation and speed etc.<br/>
+2)	The speed and the angles have been optimized as per the design and weight of the bug.<br/>
+3)	A rechargeable 11V battery has also been connected to Arduino for power supply.<br/>
+
+## Flowchart
+
+![Screenshot (20)](https://user-images.githubusercontent.com/55125159/88693423-e46f2480-d11c-11ea-87b7-0cd2659561b9.png)
+
+
+## Cost Structure:
+|Components    |Number Used|Cost(INR.) |
+|---           |---	   |---        |
+|Arduino UNO   |1          |500        |
+|Battery       |1          |400        |
+|Caster Wheel  |1          |25         |
+|Servo Motors|3 |250  |
+|Bluetooth Module |1   |350   |
+|Miscellaneous   |---|150 |
+|Total |2175|
+	
+                                                            
+
+## Application:
+<p align="justify">
+1)	Making surveillance of any disaster-affected area where human beings can’t go.<br/>
+2)	Some areas need to be placed under video surveillance for a short time only, and therefore equipping them with stationary CCTV systems is not viable. Such challenges often arise in agricultural settings. For example, the fields might only need to be placed under video surveillance when the crops are reaching maturity. Surveillance of pastures is advantageous only while cattle are roaming around them. <br/>
+3)	Can be used in industries to detect parts of the machinery that are faulted.<br/> 
+
+## Limitations:
+<p align="justify">
+1)	The speed of a robot is a concern. It can not to be used for detection purposes when time is a major constraint.<br/>
+2)	It is occupying more space than expected due to use of macro Arduino, 11V battery.<br/>
+3)	It currently has only recorded videos and images and not live recording.<br/>
+
+## Further Improvements:
+<p align="justify">
+1)	We can use nano Arduino, a battery of 9V (smaller than 11V) hence reducing surface areas, in return reducing the dimensions of legs being attached. This will help in reducing space occupied and also increasing the speed due to lesser weight load on motors.<br/>
+2)	We can also include the features of live recording by using RasberryPi.<br/>
+
+
+
+## Team Members:
+1)	Himank Sehgal
```

Comment:
Remove these unnecessary spaces and make it a proper ordered list. Also, hyperlink names with their GitHub profile link.